lib/hanami/config/logger.rb

```diff
@@ -89,8 +120,16 @@ def app_name(value = nil)
       # @see Hanami::Config::Logger#stream
       # @see Hanami::Config::Logger#engine
       def build
-        @engine ||
-          ::Hanami::Logger.new(@app_name, device: @device)
+        logger = ::Hanami::Logger.new(@app_name, device: @device)
+        logger.level = @level
+
+        @engine || logger
+      end
+
+    private
+
+      def detect_level(level)
+        level.is_a?(Integer) ? level : LEVEL_HASH[level.to_sym.downcase]
```

Comment:
@davydovanton What happens if I pass an integer that can't be recognized as level? Eg `999`.